GSoC-2025/ethology ; support for any point trackers (Richard Dushime Mudahera).md

```diff
@@ -0,0 +1,249 @@
+      </td>
+      <th style="border: 1px solid #ddd; padding: 8px;">Continued Community Engagement</th>
+      <td style="border: 1px solid #ddd; padding: 8px;">30</td>
+    </tr>
+  </tbody>
+</table>
+
+### Key Milestones
+
+- Week 3: First end-to-end tracking prototype.
+- Week 6: Mid-term review with core features functional.
+- Week 9: Stretch goals implementation begins.
+- Week 12: Project submission with full test coverage.
+
+### Communication Plan
+- **Mentor Meetings:** Hold weekly video or voice calls to discuss progress, challenges, and next steps.
+- **Daily/Weekly Updates:** Share progress and ask quick questions on Zulip.
+- **Documentation & Issue Tracking:** Use GitHub Issues and project boards to track tasks and milestones.
+- **Code Reviews:** Submit regular or weekly pull requests to ensure the project stays on track and meets community standards.
+
+## Personal Statement
+
+### Past Experience
+I have built a good foundation in Python and computer vision by working on several open source projects. My contributions to NIU repositories show that I can work well with open source communities. While I am still learning machine learning and using PyTorch, I am eager to learn more. I have worked on projects that required both research-level and production-quality code. I have also collaborated with Research Software Engineers and participated in events like the Collaborations Workshop 2024, which has deepened my interest in software sustainability and research projects and tools.
+
+### Motivation: Why This Project?
+Animal behavior research faces challenges because many tools are too strict and do not work well with non-model organisms or new types of behaviors. Any-point tracking is a new approach that can help researchers study animals more easily by tracking any point on the animal in a video, not just fixed body parts. Ethology’s goal of unifying different computer vision tasks under one easy-to-use interface matches my belief that accessible tools can boost scientific discovery. I am excited about this project because it can help researchers ask new questions and gain better insights into animal behavior.
+
+### Match: Why Me?
+I have solid skills in Python, and I am learning more about machine learning and data visualization. My experience in writing clear and useful code, both for research and for production, makes me a good fit for this project. I am eager to learn more and learn about the napari ecosystem and to use my skills to build high-quality software that meets the needs of animal behavior research. I believe my background and passion for open-source development will help me make a positive contribution to this project.
```

Comment:
> I have solid skills in Python, and I am learning more about machine learning and data visualization

This is very relevant background, can you provide examples on how these skills were / are being acquired? For example via course work or extracurricular work...